source/documentation/information/githubfeature.html.md.erb

```diff
@@ -6,3 +6,43 @@ review_in: 6 months
 ---
 
 # GitHub Feature Requests
+
+# Making Feature Requests to GitHub
+
+At the Ministry of Justice, we encourage developers and analysts to suggest improvements and new features that could enhance their experience with GitHub. Here's how you can submit a feature request:
+
+## Step 1: Use the Issue Template
+
+Begin by navigating to the [operations-engineering](https://github.com/ministryofjustice/operations-engineering) repository and select the 'Issue' tab. Click on the 'New issue' button. Choose the `GitHub Feature Request` issue template. This template contains relevant fields for your request and will help us track it effectively.
+
+## Step 2: Define Your Feature Request
+
+Your request should take the form of a user story with defined acceptance criteria. This will help us understand the context and the value your feature would provide.
+
+```markdown
```

Comment:
````markdown` doesn't do anything. It can be ``` and still provide the same functionality

Reply:
Thanks - this was a copy/paste of the current page so will need reformatting etc